content/800-data-platform/050-data-proxy/index.mdx

```diff
@@ -0,0 +1,103 @@
+* [Connection pooling and connection management](/data-platform/data-proxy#connection-pooling-and-connection-management)
+* [Reduced bundle size](/data-platform/data-proxy#reduced-bundle-size)
+* [Cold starts prevention](/data-platform/data-proxy#cold-starts-prevention)
+
+</TopBlock>
+
+## Architecture
+
+Prisma develops and maintains the Data Proxy infrastructure and services to provide connection pooling, load balancing, scaling, and fault tolerance for database applications that run in serverless and edge environments.
+
+Data Proxy infrastructure is currently available in two regions:
+
+* US East 1 (West Virginia, United States)
+* EU Central 1 (Frankfurt, Germany)
+
+<Admonition>
+
+**Request a new Data Proxy region**<br /><br />
+
+If you want us to make Data Proxy available in a new region, you can submit your request [here](
+https://prisma103696.typeform.com/to/Y03RBE5c).
+
+</Admonition>
+
+In each region, the Data Proxy infrastructure has multiple clusters that enable the load balancing, high availability, and failover of database connection pooling and management.
+
+You can use Data Proxy with [all databases that Prisma ORM supports](/reference/database-reference/supported-databases).
+
+## Connection pooling and connection management
+
+Data Proxy provides connection pooling so that your application can reuse active connections and never exhaust the maximum number of connections that your database allows.
+
+Connection pooling with the Data Proxy utilizes orchestrated Prisma [query engine](/concepts/components/prisma-engines/query-engine) technology to open, maintain, and close connection pools and, thus, help applications load-balance and scale database requests based on demand.
+
+A Data Proxy connection pool maintains active database connections and ensures that an application reuses those when new requests come in.
+
+In each region, Data Proxy management services orchestrate the lifecycle of connection pools. The lifecycle of a Data Proxy connection pool goes through the following stages:
+
+* the assignment of connection pools from each available cluster in the region
+* the increase of connection pools when the workload of your application increases
+* the maintenance of idle connection pools
+* the decrease and stop of connection pools after six minutes of inactivity within a pool
+
+This orchestration ensures that you accumulate Data Proxy usage (in CPU milliseconds) only when your database processes queries.
+
+The orchestration of Data Proxy connection pools ensures high availability, load balancing, and scalability of your application.
+
+* **High availability**. In each region, the Data Proxy infrastructure runs multiple clusters. By default and until your application reaches higher workloads, from each cluster the Data Proxy starts and assigns a connection pool to an application. If a cluster goes through a period of downtime, the remaining ones take over the management of connection pools.
```

Comment:
I don't have a single specific suggestion, but the second sentence could maybe be a bit smoother. Maybe something like:

```suggestion
* **High availability**. In each region, the Data Proxy infrastructure runs multiple clusters. By default and until your application reaches higher workloads, the Data Proxy starts and assigns a connection pool to an application from each cluster. If a cluster goes through a period of downtime, the remaining ones take over the management of connection pools.
```

But I'm not sure if that changes the meaning. It's the proximity of the clause "and until your... workloads" and then "from each cluster" that I think makes it hard to parse, so I was trying to separate these out. The solution might be to move or recast the sentence differently (maybe move "and until your... workloads"), instead of my suggestion above.